src/app/core/dialogs/menu-dialog/menu.component.html

```diff
@@ -37,23 +37,17 @@
 
       <div>
         <div style="width: 100%;">
-          <p class="main-link">Über uns</p>
+        <a class="main-link" title="Über codetekt" (click)="navigate('/')">Über codetekt</a>
```

Comment:
Statt der `navigate` Methode könnte hier doch `[routerLink]` genutzt werden, oder?

Reply:
Mit Router-Link wird das Menü nicht geschlossen. In der Navigate-Methode ist das mit drin.
`this.dialogRef.close({ route: true });`